src/main/scala/Cqrs/Projection.scala

```diff
@@ -1,40 +1,16 @@
 package Cqrs
 
 import Cqrs.Aggregate._
-import InMemoryDb._
-
-import scala.collection.immutable.TreeMap
+import DbAdapters.InMemoryDb._
```

Comment:
TODO, referencing the actual db adapter is still bad here - extract `consumeEvents`?